.github/semantic.yml

```diff
@@ -166,7 +166,6 @@ scopes:
   - aws_ec2_metadata transform # Anything `aws_ec2_metadata` transform related
   - dedupe transform # Anything `dedupe` transform related
   - filter transform # Anything `filter` transform related
-  - geoip transform # Anything `geoip` transform related
```

Comment:
noting that this is produced via terraform in the resources repo

Reply:
https://github.com/vectordotdev/resources/pull/177